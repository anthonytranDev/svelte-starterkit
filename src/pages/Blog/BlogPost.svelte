<div data-testid="BlogPost">
  <h2>Blog Post #{$$props.params.id}</h2>
</div>
