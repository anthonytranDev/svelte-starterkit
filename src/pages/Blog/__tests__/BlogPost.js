import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/svelte"
import { Router, Route } from "svelte-routing"

import BlogPost from "../BlogPost.svelte"

describe("BlogPost", () => {
  it("should render", () => {
    const { getByTestId } = render(
      <Router>
        <BlogPost params="{params}" />
      </Router>
    )

    expect(getByTestId("BlogPost")).toMatchSnapshot()
  })
})
