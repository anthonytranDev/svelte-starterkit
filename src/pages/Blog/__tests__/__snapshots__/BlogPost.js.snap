// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost should render 1`] = `
<div
  data-testid="BlogPost"
>
  <h2>
    Blog Post #
    undefined
  </h2>
</div>
`;
